src/v2/guide/plugins.md

```diff
@@ -6,15 +6,15 @@ order: 304
 
 Plugins usually add global-level functionality to Vue. There is no strictly defined scope for a plugin - there are typically several types of plugins:
 
-1. Add some global methods or properties. e.g. [vue-custom-element](https://github.com/karol-f/vue-custom-element)
+1. Add some global methods or properties e.g. [vue-custom-element](https://github.com/karol-f/vue-custom-element)
```

Comment:
I like the fact that we're removing periods since it's a list item and only consists of one sentence. However, we do need to wrap the `e.g.` portion in a parentheses as it is an optional part of the sentence.

```
1. Add some global methods or properties (e.g. [vue-custom-element](https://github.com/karol-f/vue-custom-element))
```